src/components/screens/AvailableVehicles.tsx

```diff
@@ -56,17 +60,21 @@ export const AvailableVehicles = ({
     <>
       <SectionHeader title={texts.pointOfInterest.availableVehicles} />
 
-      {availableVehiclesData.map((item: AvailableVehiclesProps, index: number) => (
-        <Fragment key={item.id}>
-          <ListItem style={styles.container}>
-            {!!iconName && <CategoryIcon />}
-            <ListItem.Content>
-              <RegularText>{item.name}</RegularText>
-            </ListItem.Content>
-          </ListItem>
-          {index !== availableVehiclesData.length - 1 && <Divider style={styles.divider} />}
-        </Fragment>
-      ))}
+      {availableVehiclesData?.length ? (
+        availableVehiclesData.map((item: AvailableVehiclesProps, index: number) => (
+          <Fragment key={item.id}>
```

Comment:
what exactly does the fragment component do here?

Reply:
same as <> but named to be able to set the key prop